### PR TITLE
feat: add http headers

### PR DESCRIPTION
**Pull request by BenediktBertsch**
Add support for http headers, fixes feature request #196.
Credits to @forthal for helping me out.

### CODE REVIEW

File: cli/cobra.go [hunk 1]
```diff
@@ -59,7 +59,10 @@ func Client(apiUlr, userID, password, otp string) (c *proxmox.Client, err error)
 	if otp == "" {
 		otp = os.Getenv("PM_OTP")
 	}
-	c, err = proxmox.NewClient(apiUlr, nil, tlsconf, proxyUrl, timeout)
+	if http_headers == "" {
+		otp = os.Getenv("HTTP_HEADERS")
```
Comment (mleone87) [inline cli/cobra.go]: error?

Comment (BenediktBertsch) [reply cli/cobra.go]: Yeah, overlooked it my bad.

File: cli/cobra.go [hunk 2]
```diff
@@ -59,7 +59,10 @@ func Client(apiUlr, userID, password, otp string) (c *proxmox.Client, err error)
 	if otp == "" {
 		otp = os.Getenv("PM_OTP")
 	}
-	c, err = proxmox.NewClient(apiUlr, nil, tlsconf, proxyUrl, timeout)
+	if http_headers == "" {
+		http_headers = os.Getenv("HTTP_HEADERS")
```
Comment (mleone87) [inline cli/cobra.go]: I would use PM_HTTP_HEADERS for coherence with other envs, what do you think?

Comment (BenediktBertsch) [reply cli/cobra.go]: Works out for me.

**Comment by mleone87** [comment]
I just ran this PR code with no PM_HTTP_HEADERS set and it does not work

`error the value of key (Header key(s) and value(s) not even. Check your PM_HTTP_HEADERS env.) may not be empty`

**Pull request merged**